Sources/Internal/Version.swift

```diff
@@ -16,5 +16,41 @@ import Foundation
 
 @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
 struct Version {
-  static let version = "11.3.0-beta"
+  static let sdkVersion = "11.3.0-beta"
+
+  // returns value of form gl-PLATFORM_NAME/PLATFORM_VERSION
+  static func platformVersionHeader() -> String {
```

Comment:
Any reason not to use https://github.com/google/GoogleUtilities/blob/main/GoogleUtilities/Environment/GULAppEnvironmentUtil.m#L195 like other Firebase products?

Reply:
I switched to using it - definitely a better option. Had missed that this existed.